zookeeper-docs/src/main/resources/markdown/zookeeperProgrammers.md

```diff
@@ -1509,10 +1476,6 @@ ZooKeeper users fall into:
 Outside the formal documentation, there're several other sources of
 information for ZooKeeper developers.
 
-* *ZooKeeper Whitepaper _[tbd: find url]_* :
```

Comment:
We could add the white paper here?
https://www.usenix.org/legacy/event/usenix10/tech/full_papers/Hunt.pdf
Not sure if that's entirely legal though, but there's als the researchGate link:
https://www.researchgate.net/publication/228366039_ZooKeeper_Wait-free_Coordination_for_Internet-scale_Systems

Anyway, I'm also fine with just removing it.

Reply:
I don't disagree, but I'm not sure the best way to include it.  Can we simply include the PDF as a resource in the ZK documentation project itself so that there need be no concern about a dead link?

I propose that this empty reference simply be deleted for now and that you open a new JIRA for figuring out the best way to add it in.

Reply:
@phunt Are you able to provide some details on this?